### PR TITLE
add PySpark

### PR DESCRIPTION
**Pull request by lsgrep**
- add PySpark

### CODE REVIEW

File: .github/workflows/bake-pyspark.hcl [hunk 1]
```diff
@@ -0,0 +1,10 @@
+target "docker-metadata-action" {}
+
+target "bake-platform" {
+  inherits = ["docker-metadata-action"]
+  context = "jupyter-pyspark"
+  dockerfile = "cpu.Dockerfile"
```
Comment (xin-hao-awx) [inline .github/workflows/bake-pyspark.hcl:6]: This should be `Dockerfile`?

Comment (yusup-airwallex) [reply .github/workflows/bake-pyspark.hcl]: Thanks. fixed.

Comment (lsgrep) [reply .github/workflows/bake-pyspark.hcl]: Thanks. fixed.

**Review by haoxins — APPROVED** [review]

**Comment by haoxins** [comment]
maybe we can base on our `jupyter` base image in the future.
after the jupyter v4 released

**Pull request merged**